From b714093632c6cb0f7855026f76d61838fb959af5 Mon Sep 17 00:00:00 2001
From: "jing.zhang" <jing.zhang@amlogic.com>
Date: Mon, 22 Nov 2021 10:54:04 +0800
Subject: [PATCH] audio: for remove log  [1/1]

PD#SWPL-62301

Problem:
remove log

Solution:
remove log

Verify:
ah212

Change-Id: Iad34edcba9a4ad56237750d536a148c8d7326be8
Signed-off-by: jing.zhang <jing.zhang@amlogic.com>
---
 dtv_audio_utils/audio_data_read/dmx_audio_es.cpp    | 4 ++--
 dtv_audio_utils/dmxwrap/MultiHwDemux/AmLinuxDvb.cpp | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/dtv_audio_utils/audio_data_read/dmx_audio_es.cpp b/dtv_audio_utils/audio_data_read/dmx_audio_es.cpp
index d7c63b7..5a2cef3 100644
--- a/dtv_audio_utils/audio_data_read/dmx_audio_es.cpp
+++ b/dtv_audio_utils/audio_data_read/dmx_audio_es.cpp
@@ -161,8 +161,8 @@ AM_Dmx_Audio_ErrorCode_t Get_MainAudio_Es(void *demux_handle,struct mAudioEsData
     if ((*mAudioEsData)->size == 0) {
         return AM_AUDIO_Dmx_ERROR;
     }
-    if (aml_audio_get_debug_flag() > 1)
-        ALOGV("\n mEsdata->pts : %lld size:%d \n",(*mAudioEsData)->pts,(*mAudioEsData)->size);
+    //if (aml_audio_get_debug_flag() > 1)
+    //    ALOGV("\n mEsdata->pts : %lld size:%d \n",(*mAudioEsData)->pts,(*mAudioEsData)->size);
 
     return (AM_Dmx_Audio_ErrorCode_t)ret;
 }
diff --git a/dtv_audio_utils/dmxwrap/MultiHwDemux/AmLinuxDvb.cpp b/dtv_audio_utils/dmxwrap/MultiHwDemux/AmLinuxDvb.cpp
index ad9c5cb..68f4101 100644
--- a/dtv_audio_utils/dmxwrap/MultiHwDemux/AmLinuxDvb.cpp
+++ b/dtv_audio_utils/dmxwrap/MultiHwDemux/AmLinuxDvb.cpp
@@ -217,7 +217,7 @@ AM_ErrorCode_t AmLinuxDvd::dvb_poll(AM_DMX_Device *dev, AM_DMX_FilterMask_t *mas
             fds[cnt].fd     = dmx->fd[i];
             fids[cnt] = i;
             cnt++;
-            ALOGV("dvb_poll %d",cnt);
+            //ALOGV("dvb_poll %d",cnt);
         }
     }
 
@@ -243,7 +243,7 @@ AM_ErrorCode_t AmLinuxDvd::dvb_poll(AM_DMX_Device *dev, AM_DMX_FilterMask_t *mas
         if (fds[i].revents&(POLLIN|POLLERR))
         {
             AM_DMX_FILTER_MASK_SET(mask, fids[i]);
-            ALOGV("fids[i] %d mask %d",fids[i],*mask);
+           // ALOGV("fids[i] %d mask %d",fids[i],*mask);
         }
     }
 
-- 
2.31.0

